Loaded .env file from: D:\Scripts\analyse\Telemark\Python\token.env
X_FUNCTIONS_KEY loaded successfully.
Loaded .env file from: D:\Scripts\analyse\Telemark\Python\token.env
GITHUB_TOKEN loaded successfully.
Flyktninger og arbeidsinnvandrere CSV data loaded successfully.
Selected data from year 2024-01-01 00:00:00
Saved file to D:\Scripts\analyse\Telemark\Python\Temp\andel_arbeidsinnvandrere.csv
[2025-01-23 15:34:28] No new data to upload. Skipping GitHub update.
Keeping CSV file: D:\Scripts\analyse\Telemark\Python\Temp\andel_arbeidsinnvandrere.csv
Saved file to D:\Scripts\analyse\Telemark\Python\Temp\andel_flyktninger.csv
[2025-01-23 15:34:28] Changes detected in the dataset.
File uploaded successfully: Data/09_Innvandrere og inkludering/Innvandrerbefolkningen/andel_flyktninger.csv

Showing up to 5 examples of changes:

Row: Label: KragerÃÂ¸ | Kommune: KragerÃÂ¸
  Andel flyktninger 2024: 2 -> 44

Row: Label: Tokke | Kommune: Tokke
  Andel flyktninger 2024: 2 -> 41

Row: Label: Vinje | Kommune: Vinje
  Andel flyktninger 2024: 2 -> 49
Email notifications disabled. Updates for andel_flyktninger.csv were not sent.
Keeping CSV file: D:\Scripts\analyse\Telemark\Python\Temp\andel_flyktninger.csv
No new data detected in andel_arbeidsinnvandrere.csv.
New data detected in andel_flyktninger.csv and pushed to GitHub.
New data status log written to D:\Scripts\analyse\Telemark\Python\Log\new_data_status_Innvandrere_-_Flyktninger_og_arbeidsinnvandrere.log


Errors/Warnings:
D:\Scripts\analyse\Telemark\Python\Queries/09_Innvandrere_og_inkludering/Innvandrerbefolkningen/andel_flyktninger_og_arbeidsinnvandrere.py:154: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_pivot_arbeidsinnvandrere["Label"] = df_pivot_arbeidsinnvandrere["Kommune"]
D:\Scripts\analyse\Telemark\Python\Queries/09_Innvandrere_og_inkludering/Innvandrerbefolkningen/andel_flyktninger_og_arbeidsinnvandrere.py:155: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df_pivot_flyktninger["Label"] = df_pivot_flyktninger["Kommune"]

